packages/geo/src/lib/datasource/shared/datasources/wms-datasource.ts

```diff
@@ -82,12 +82,30 @@ export class WMSDataSource extends DataSource {
       options.sourceFields = [];
     }
 
```

Comment:
Please make a variable out of "(this.options as OgcFilterableDataSourceOptions).ogcFilters". This will make the code muich more readable and concise.